crates/workers/src/workers.rs

```diff
@@ -30,7 +33,25 @@ pub struct Workers {
     /// Key storage for managing worker key pairs.
     key_storage: Arc<KeyStorage>,
     /// Scope information used to determine the host and manage key pairs.
-    scope: Scope,
+    scope: PeerScope,
+    /// Mapping of worker IDs to worker runtime.
+    runtimes: RwLock<HashMap<WorkerId, Runtime>>,
```

Comment:
```suggestion
    thread_pools: RwLock<HashMap<WorkerId, Runtime>>,
```
maybe?

not sure how else to convey that it's the tokio runtime, and it's about threads. Otherwise it might be Docker runtime or WASM runtime, or something else.